src/gen_bpf.c

```diff
@@ -45,6 +46,11 @@
 #define AINC_BLK			2
 #define AINC_PROG			64
 
+/* binary tree definitions */
+#define SYSCALLS_PER_NODE		(4)
+#define INVALID_HSH			(UINT64_MAX)
+#define INVALID_SYSCALL			(UINT_MAX)
```

Comment:
I might preface these with something that would indicate they are specific to the binary tree optimization, especially since we use hashing for other BPF block optimizations as well.

Maybe `BTREE_HSH_INVALID`, etc.?